src/quick-start.md

```diff
@@ -2,45 +2,59 @@
+
+## Installation
+
+> **PREREQUISITES:**
+>
+> Currently, the chart depends on cert-manager. Make sure you have [`cert-manager` installed](https://cert-manager.io/docs/installation/) *before* installing the kubewarden-controller chart.
+>
+> You can install the latest version of `cert-manager` by running the following commands: 
+>
+```shell
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
 
-## Install
+kubectl wait --for=condition=Available deployment --timeout=2m -n cert-manager --all
+```
 
-The Kubewarden stack can be deployed using a helm chart.
-Currently, the chart depends on cert-manager. Make sure you have [`cert-manager` installed](https://cert-manager.io/docs/installation/) and then install the kubewarden-controller chart. For example:
+The Kubewarden stack can be deployed using a `helm` charts as follow:
 
-```console
+```shell
 helm repo add kubewarden https://charts.kubewarden.io
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
-kubectl wait --for=condition=Available deployment --timeout=2m -n cert-manager --all
+
 helm install --wait -n kubewarden --create-namespace kubewarden-crds kubewarden/kubewarden-crds
+
 helm install --wait -n kubewarden kubewarden-controller kubewarden/kubewarden-controller
 ```
 
-This will install on the Kubernetes cluster:
-* `kubewarden-crds`  will register the
-`ClusterAdmissionPolicy` and `PolicyServer` Custom Resources.
+The following components should be installed inside the `kubewarden` namespace in your Kubernetes cluster:
+
+* `kubewarden-crds` which will register the
+`ClusterAdmissionPolicy` and `PolicyServer` Custom Resources
 
 * `kubewarden-controller` with
-the default configuration. It will create a `PolicyServer` resource named `default`. The components of the
-Kubewarden stack will be deployed inside of a Kubernetes Namespace
-called `kubewarden`.
+a default configuration and which will create a `PolicyServer` resource named `default`. 
+
+> **QUICK NOTES:**
+>
+> The default configuration values should be good enough for the majority of deployments, and all the options are documented [here](https://charts.kubewarden.io/#configuration).
+
+## Main components
 
-The default configuration values should be good enough for the majority of
-deployments, all the options are documented [here](https://charts.kubewarden.io/#configuration).
+Kubewarden has two main components which you will interact with:
+* The Policy Server
+* The ClusterAdmissionPolicy
 
-The Kubewarden Policy Servers are completely managed by the `kubewarden-controller`.
+### Policy Server
 
-## Policy Server
+A Kubewarden Policy Server is completely managed by the `kubewarden-controller` and multiple Policy Servers can be deployed in the same Kubernetes cluster.
 
-Represents a Deployment of Kubewarden `policy-server`, which receives the requests to be validated. It does that
```

Comment:
Here we were talking about the binary that gets run in a Deployment, if a PolicyServer is created. I think it was confusing, as there's not much to be gained from that info in the quickstart. I propose we substitute all `policy-server` instances with `PolicyServer`. We can keep this line or similar, explaining that instantiating a `PolicyServer` will create a Deployment with the server, so people know what to expect.